src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -81,6 +81,29 @@ public enum ReminderServiceProviderType
         }
 
         /// <summary>
+        /// Configuration for Gossip Channels
+        /// </summary>
+        public enum GossipChannelType
+        {
+            /// <summary>Default value to allow discrimination of override values.</summary>
+            NotSpecified,
```

Comment:
`NotSpecified` is what is currently used by `LivenessProviderType` and `ReminderServiceProviderType` (right above it in same file). To be consistent, I will leave it the same, unless you think we should change all of them.